### ISSUE TITLE
pa_2 done

### ISSUE THREAD

**Issue by GaboSJW** [GaboSJW/pa_2_2022#1]
@jvcasillas

**Comment by jvcasillas** [comment]
See point 7 of section 2.2

**Comment by jvcasillas** [comment]
I don't think you knit the document, so I didn't see any of the plots you made, but everything else is good to go. Also, did you see my comment above? @GaboSJW 

|                 Task                             | Points |     |
| :----------------------------------------------- | -----: | ---:|
| Create the described project structure           |    0.5 | 0.5    |
| Create master stim wav and textgrid files        |    0.5 | 0.5    |
| Segment master file for individual items         |      1 | 1    |
| Correctly segment individual items               |      1 | 1    |
| Fix loop and extract data                        |      1 | 1    |
| Create an RStudio project                        |      1 | 1    |
| Create an Rmarkdown README file                  |      1 | 1    |
| Calculate duration, intensity and F0 averages    |      3 | 3    |
| Create `pa_2` repo                               |    0.5 | 0.5    |
| Commit and push to repo, tag me                  |    0.5 | 0.5    |
| **Total**                                        |     10 | 10    |

**Comment by GaboSJW** [comment]
> I don't think you knit the document, so I didn't see any of the plots you made, but everything else is good to go. Also, did you see my comment above? @GaboSJW
> 
> Task	Points	
> Create the described project structure	0.5	0.5
> Create master stim wav and textgrid files	0.5	0.5
> Segment master file for individual items	1	1
> Correctly segment individual items	1	1
> Fix loop and extract data	1	1
> Create an RStudio project	1	1
> Create an Rmarkdown README file	1	1
> Calculate duration, intensity and F0 averages	3	3
> Create `pa_2` repo	0.5	0.5
> Commit and push to repo, tag me	0.5	0.5
> **Total**	10	10

**Comment by jvcasillas** [comment]
Still not sure you read point 7 of section 2.2 ;) @GaboSJW

**Comment by GaboSJW** [comment]
> I don't think you knit the document, so I didn't see any of the plots you made, but everything else is good to go. Also, did you see my comment above? @GaboSJW
> 
> Task	Points	
> Create the described project structure	0.5	0.5
> Create master stim wav and textgrid files	0.5	0.5
> Segment master file for individual items	1	1
> Correctly segment individual items	1	1
> Fix loop and extract data	1	1
> Create an RStudio project	1	1
> Create an Rmarkdown README file	1	1
> Calculate duration, intensity and F0 averages	3	3
> Create `pa_2` repo	0.5	0.5
> Commit and push to repo, tag me	0.5	0.5
> **Total**	10	10

@jvcasillas 
I didn't see the comment above. 
I checked Point 7 and Section 2.2 and did it all over again with a new Rmd file and knitted again.

In the first push, I included the knitted file, but when I checked the _code_ tab, the preview for [README.html](https://github.com/GaboSJW/pa_2#readme) was really messy, so I deleted it. Now I knitted again, and the file still looks ugly in preview. Is that normal?
<img width="928" alt="Screen Shot 2022-03-21 at 23 40 29" src="https://user-images.githubusercontent.com/45706205/159403378-40e52cfb-65cc-419d-a56d-7ad11fa0ef5a.png">

**Comment by jvcasillas** [comment]
Yeah, that's not what it should look like. I'm referring to this part here: 

> Create a README.Rmd file and set the output as **github_document** (see pa_1 if you don’t remember). Save it in the root directory of pa_2.

**Comment by GaboSJW** [comment]
@jvcasillas
Ooops... I see what you mean now...
Just pushed again
Thank you for the hint!

**Comment by jvcasillas** [comment]
👍🏼
